src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,11 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data. The following errors has occured: %s\n";
+
+    public static final String FILE_OPS_PERMISSION_ERROR_MESSAGE = "Could not save data to file: "
+            + "Insufficient permissions to write content to file %s.\n";
```

Comment:
```suggestion
    public static final String FILE_OPS_PERMISSION_ERROR_MESSAGE = "Could not save data to file. "
            + "Insufficient permissions to write to the file %s.\n";
```
* Made punctuation more consistent.
* Removed redundant word `content`